relay-event-schema/src/protocol/security_report.rs

```diff
@@ -1069,7 +1135,9 @@ impl SecurityReportType {
     pub fn from_json(data: &[u8]) -> Result<Option<Self>, serde_json::Error> {
         #[derive(Deserialize)]
         #[serde(rename_all = "kebab-case")]
-        struct SecurityReport {
+        struct SecurityReport<'a> {
+            #[serde(rename = "type", borrow)]
+            ty: Option<&'a str>,
```

Comment:
This is going to fail deserialization when there is anything escaped in the type. Should use a `Cow<'a, str>`